src/assets/css/main.css

```diff
@@ -32,11 +38,29 @@
 p {
   font: 1.2em/1.62 sans-serif;
 }
-.join-banner {
-  width: 100%;
-  height: 100px;
+.join-icon {
+  width: 80px; 
+  height: 80px; 
   object-fit: scale-down;
 }
+.join-text {
+  display: inline-block; 
+  margin: 0 10px 0 10px;
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+}
+.join-header {
+  display: flex;
+  flex-direction: row;
+}
+.join-title {
+  margin: 0px;
+  flex: 1;
+}
+.join-desc {
+  font-size: 0.9em;
+}
```

Comment:
There should be very little custom css here, I'm already using a responsive css framework: https://jenil.github.io/chota/

The flex stuff especially shouldn't be necessary.

Reply:
I tried to make it working with that, but the whole css layout logic makes no sense to me, especially with the table-based stuff. Not to mention making it scale to mobile. So i dont think i will be able to implement this much better.